util.py

```
@@ -79,9 +79,9 @@ def train_one_epoch(cfg, model, loss_ce, scheduler, optimizer, epochID, max_epoc
     lossTrainNorm = 0
 
     scheduler.step()
-    pbar = tqdm(total=cfg['bath_per_epoch'])
+    pbar = tqdm(total=cfg['batch_per_epoch'])
     for batchID, (index, input, target, binary_label) in enumerate(train_loader):
```

Ruff (0.14.4)

Loop control variable `index` not used within loop body

(B007)